client/state/selectors/is-gutenberg-enabled.js

```diff
@@ -1,44 +1,30 @@
 /** @format */
 
-/**
- * External dependencies
- */
-import { get } from 'lodash';
-
 /**
  * Internal dependencies
  */
-import { isEnabled } from 'config';
-import getSiteOptions from 'state/selectors/get-site-options';
 import isCalypsoifyGutenbergEnabled from 'state/selectors/is-calypsoify-gutenberg-enabled';
-import isVipSite from 'state/selectors/is-vip-site';
-import { isJetpackSite } from 'state/sites/selectors';
-import versionCompare from 'lib/version-compare';
+import isGutenframeEnabled from 'state/selectors/is-gutenframe-enabled';
+import isVipSite from './is-vip-site';
 
 export const isGutenbergEnabled = ( state, siteId ) => {
 	if ( ! siteId ) {
 		return false;
 	}
 
+	if ( isVipSite( state, siteId ) ) {
+		return false;
+	}
+
 	if ( isCalypsoifyGutenbergEnabled( state, siteId ) ) {
```

Comment:
I think we should be able to remove this function too, and transfer some parts of it to `isGutenbergEnabled` where needed (e.g. WP version check). In general I think it would be good if we could remove all references to "calypsoify" in Gutenberg to avoid confusion.